src/pages/ProfilePage.jsx

```diff
@@ -0,0 +1,9 @@
+import JoinedList from '../components/JoinedList/JoinedList';
+
+const ProfilePage = () => (
+  <div className="profile">
+    <JoinedList />
```

Comment:
Awesome 🔥 You are rendering a list of reserved rockets and missions without any issue! 

![image](https://user-images.githubusercontent.com/15807343/167960484-3149e1d7-d403-4611-90a7-41f1a23cf2cf.png)